tests/hla/test_randomlist.py

```diff
@@ -168,8 +179,10 @@ def align_randomFields(self, randomTable):
                 continue
 
         # Perform some clean up
-        if os.path.exists('ref_cat.ecsv'): os.remove('ref_cat.ecsv')
-        if os.path.exists('refcatalog.cat'): os.remove('refcatalog.cat')
+        if os.path.exists('ref_cat.ecsv'): 
```

Comment:
I recommend using `os.path.isfile()` instead of the generic `os.path.exists`. It is more specific.

Reply:
OK fixed.